Content.Shared/Doors/Components/DoorComponent.cs

```diff
@@ -273,41 +291,102 @@ private float? SecondsUntilStateChange
     /// <summary>
     /// Whether the door blocks light.
     /// </summary>
-    [ViewVariables(VVAccess.ReadWrite)]
     [DataField]
     public bool Occludes = true;
 
     /// <summary>
     /// Whether the door will open when it is bumped into.
     /// </summary>
-    [ViewVariables(VVAccess.ReadWrite)]
     [DataField]
     public bool BumpOpen = true;
 
     /// <summary>
     /// Whether the door will open when it is activated or clicked.
     /// </summary>
-    [ViewVariables(VVAccess.ReadWrite)]
     [DataField]
     public bool ClickOpen = true;
 
     [DataField(customTypeSerializer: typeof(ConstantSerializer<DrawDepthTag>))]
-    public int OpenDrawDepth = (int) DrawDepth.DrawDepth.Doors;
+    public int OpenDrawDepth = (int)DrawDepth.DrawDepth.Doors;
 
     [DataField(customTypeSerializer: typeof(ConstantSerializer<DrawDepthTag>))]
-    public int ClosedDrawDepth = (int) DrawDepth.DrawDepth.Doors;
+    public int ClosedDrawDepth = (int)DrawDepth.DrawDepth.Doors;
+
+    // /// <summary>
+    // /// Is true if the door is currently opening or closing due to being pried.
+    // /// </summary>
+    // [DataField, AutoNetworkedField, ViewVariables(VVAccess.ReadOnly)]
+    // public bool IsBeingPried;
 }
 
 [Serializable, NetSerializable]
 public enum DoorState : byte
 {
+    /// <summary>
```

Comment:
Meaningful tech change two: revised the door states to cover all the states a door can be in, rolling in the "partial" and "pried" bools. This means we can orient code around states alone. This makes code much less fragile.